receiver/hostmetricsreceiver/internal/scraper/diskscraper/disk_scraper_test.go

```diff
@@ -36,29 +36,20 @@ func TestScrapeMetrics(t *testing.T) {
 		// expect 3 metrics
 		assert.Equal(t, 3, metrics.Len())
 
-		// for disk byts metric, expect a receive & transmit datapoint for at least one drive
-		hostDiskBytesMetric := metrics.At(0)
-		internal.AssertDescriptorEqual(t, metricDiskBytesDescriptor, hostDiskBytesMetric.MetricDescriptor())
-		assert.GreaterOrEqual(t, hostDiskBytesMetric.Int64DataPoints().Len(), 2)
-		internal.AssertInt64MetricLabelHasValue(t, hostDiskBytesMetric, 0, directionLabelName, receiveDirectionLabelValue)
-		internal.AssertInt64MetricLabelHasValue(t, hostDiskBytesMetric, 1, directionLabelName, transmitDirectionLabelValue)
-
-		// for disk operations metric, expect a receive & transmit datapoint for at least one drive
-		hostDiskOpsMetric := metrics.At(1)
-		internal.AssertDescriptorEqual(t, metricDiskOpsDescriptor, hostDiskOpsMetric.MetricDescriptor())
-		assert.GreaterOrEqual(t, hostDiskOpsMetric.Int64DataPoints().Len(), 2)
-		internal.AssertInt64MetricLabelHasValue(t, hostDiskOpsMetric, 0, directionLabelName, receiveDirectionLabelValue)
-		internal.AssertInt64MetricLabelHasValue(t, hostDiskOpsMetric, 1, directionLabelName, transmitDirectionLabelValue)
-
-		// for disk time metric, expect a receive & transmit datapoint for at least one drive
-		hostDiskTimeMetric := metrics.At(2)
-		internal.AssertDescriptorEqual(t, metricDiskTimeDescriptor, hostDiskTimeMetric.MetricDescriptor())
-		assert.GreaterOrEqual(t, hostDiskTimeMetric.Int64DataPoints().Len(), 2)
-		internal.AssertInt64MetricLabelHasValue(t, hostDiskTimeMetric, 0, directionLabelName, receiveDirectionLabelValue)
-		internal.AssertInt64MetricLabelHasValue(t, hostDiskTimeMetric, 1, directionLabelName, transmitDirectionLabelValue)
+		// for disk byts metric, expect a read & write datapoint for at least one drive
```

Comment:
I personally prefer the test conditions in the test case, but not strong opinion on this.